learn/guides/graphql-client-tool.md

```diff
@@ -14,69 +14,156 @@ redirect_from:
   - /learn/graphql-client-tool
 --- 
 
-GraphQL schemas of a service are now most often specified using what’s known as the GraphQL SDL (schema definition language). It is also sometimes referred to as just GraphQL schema language. Ballerina GraphQL client tool will make it easy for you to generate a client in Ballerina given the GraphQL schema (SDL) and GraphQL queries. You can generate multiple clients in Ballerina for multiple GraphQL documents for a given GraphQL SDL. It also enables you to generate multiple Ballerina modules for multiple GraphQL projects to work with different GraphQL APIs. 
+The GraphQL schema is necessary to validate the GraphQL queries. Each GraphQL query will be validated against the given GraphQL schema before generating a Ballerina client remote operation and the corresponding Ballerina record types. Use the generated Ballerina client to execute queries/mutations on the relevant GraphQL API through the generated remote operations.
```

Comment:
This sentence is not clear.

Reply:
When generating the client, this tool validates the given GraphQL queries against the given GraphQL schema. A Ballerina client remote operation and the corresponding Ballerina record types will be generated for each given GraphQL query. The generated Ballerina client can be used to execute queries (i.e., that belong to one of the operation types, `query` or `mutation`) on the relevant GraphQL API through the generated remote operations.

Reply:
@dakshitha - Can you check this?

Reply:
A Ballerina client remote operation - why do we have to say Ballerina client remote operation? Can't it be a remote operation? I don't get the client remote operation part. Or the client's remote operation? Or the remote operation of the client?

Reply:
Updated. Thank you.